scripts/terraform-apply.sh

```diff
@@ -11,16 +11,14 @@ set -e
 # are exposed via terraform apply.
 
 # Make redact-output.sh executable
-chmod +x ./scripts/redact-output.sh
+chmod +x $(dirname $0)/redact-output.sh
```

Comment:
Given that this script will always run from this directory, is there any benefit from using `dirname` to derive the directory name? Does an absolute path to the `redact-output.sh` offer any benefits?

Reply:
In this pipeline, I'm calling this from the terraform project instead, like this for example.  So I don't need to be forever changing directory, e.g. with the workspace select.

      - name: Init and State Refresh
        working-directory: "${{ env.TEAM_DIR }}/${{ matrix.project }}"
        run: |
          ${GITHUB_WORKSPACE}/scripts/terraform-init.sh .
          terraform workspace select "core-shared-services-${TF_ENV}"
          ${GITHUB_WORKSPACE}/scripts/terraform-refresh.sh .